helm/dashboards/Chart.yaml

```diff
@@ -4,8 +4,7 @@ engine: gotpl
 home: https://github.com/giantswarm/dashboards
 icon: https://s.giantswarm.io/app-icons/grafana/1/light.svg
 name: dashboards
-appVersion: [[ .AppVersion ]]
```

Comment:
appVersion is entirely removed ? I guess it makes sense for dashboard though :thinking:

Reply:
Yes we do not have an app version